.travis.yml

```diff
@@ -9,24 +9,27 @@ script:
   - ruby script/jack_hammer -t 2000
 matrix:
   include:
-    - rvm: 2.1.8
+    - rvm: 2.2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-3.2.x
       env: DB=mysql
-    - rvm: 2.1.8
+    - rvm: 2.1.9
       gemfile: spec/support/gemfiles/Gemfile.rails-4.2.x
       env: DB=mysql
-    - rvm: 2.2.4
-      gemfile: spec/support/gemfiles/Gemfile.rails-4.2.x
+    - rvm: 2.2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.0.x
       env: DB=mysql
-    - rvm: 2.3.0
+    - rvm: 2.3.1
       gemfile: spec/support/gemfiles/Gemfile.rails-4.1.x
```

Comment:
Yes, we should remove them eventually. But for now let's leave them in as a favour to all the companies running large monolithic applications on legacy Rails versions and having trouble upgrading. 😉